src/api/app/models/bs_request_action.rb

```diff
@@ -400,6 +400,7 @@ def source_cleanup
     Package.source_path(self.source_project, self.source_package)
   end
 
+  # rubocop:disable  Metrics/AbcSize
```

Comment:
if you need to touch it once more, remove that space please

Reply:
done!